docs/pages/eas/metadata/config.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Create config files for EAS Metadata
+sidebar_title: Creating the store config
```

Comment:
"Store config" seems a little bit more ambiguous since it's more of a "Metadata config" (also seems ok). One thing I like about `store.config.json` is that we always use it for downloading, and it implies this is a JSON-related data structure.

Reply:
I changed all references to the `store.config.json` to be the term "store config", since we also support 2 different types of "store configs"; `store.config.json` and `store.config.js`. It's just a consistency thing, we can update that if needed.

For the schema, I think we can name that `Metadata Schema`, since that refers to the object within the store config.

Reply:
I'll update the sidebar links with this ^